models/utilities/default_model_mod.f90

```diff
@@ -49,8 +51,24 @@ module default_model_mod
           convert_vertical_obs,   &
           convert_vertical_state, &
           read_model_time,        &  ! from the dart_time_io module
-          write_model_time
-
+          write_model_time,       &
+          parse_variables,        &
+          parse_variables_clamp,  &
+          state_var_type,         &
+          MAX_STATE_VARIABLE_FIELDS, &
+          MAX_STATE_VARIABLE_FIELDS_CLAMP
+
+type :: state_var_type
+    integer                        :: nvars
+    character(len=NF90_MAX_NAME), allocatable :: netcdf_var_names(:)
+    integer, allocatable           :: qtys(:)
+    real(r8), allocatable          :: clamp_values(:, :)
+    logical, allocatable           :: updates(:)
+end type state_var_type
+
```

Comment:
there is a relationship between these parameters that needs to remain the same if MAX_STATE_VARIABLES is increased. 

You can't code this, since they are parameters, so note in comment

MAX_STATE_VARIABLE_FIELDS = MAX_STATE_VARIABLES*3
MAX_STATE_VARIABLE_FIELDS_CLAMP = MAX_STATE_VARIABLES*5

Reply:
added comment in d43f7ee13b583300340b0d86de863063d139c292